src/02-common-principles.md

```diff
@@ -48,9 +48,15 @@ misunderstanding we clarify them here.
     subject performed multiple conceptually different behaviors (with different
     sets of instructions) they will be considered one (combined) task.
 
-1.  **Event** - a stimulus or subject response recorded during a task. Each
-    event has an onset time and duration. Note that not all tasks will have
-    recorded events (e.g., “resting state”).
+1.  **Event** - something that happens or may be perceived as happening at a particular
+    moment during the recording.
+    Events are most commonly associated with sudden-onset sensory stimulus presentations
+    or with the distinct onset of a subject motor action.
+    Other events may include unplanned incidents,
+    changes in task instructions or experiment control parameters,
+    and noted data feature occurrences.
```

Comment:
```suggestion
    Other events may include unplanned incidents (e.g. sudden onset of noise and vibrations due to construction works, laboratory device malfunction etc.),
    changes in task instructions (e.g., switching the response hand) or experiment control parameters,
    and noted data feature occurrences.
```

Reply:
I'm not sure what "changes in … experiment control parameters" is supposed to mean, do we have an example?

Same for "noted data feature occurrences" – you mean something like, "the experimenter realized one channel started to produce noise", this kind of occurrence?

Reply:
> something like, "the experimenter realized one channel started to produce noise", this kind of occurrence?

this is an example that seems to fall under the currently proposed definition, yes.

> I'm not sure what "changes in … experiment control parameters" is supposed to mean, do we have an example?

maybe the [eeg_matchingpennies](https://github.com/bids-standard/bids-examples/blob/master/eeg_matchingpennies/sub-05/eeg/sub-05_task-matchingpennies_events.tsv) dataset is an example for this ... we were doing a BCI game where the participant tried to outplay a computer reading the participant's EEG. over the course of the recording, the "experiment parameter" of BCI training changed (stage 1=bci classifier not trained, stage 2 and 3, classifier trained on data from previous stages) --> :man_shrugging:

Reply:
A common example of "changes in experimental control parameters" comes from experiments organized into blocks during which the control variables are fixed for a series of trials. Suppose one of the control variables is the presentation rate, which is either "slow" or "fast". A given session might have two counter-balanced blocks , one block has the "slow" rate and the other the fast rate.  The starting and stopping points of each block are typically marked with events.  This information is extremely valuable for analysts.

Reply:
I think that  "Events are most commonly associated with sudden-onset sensory stimulus presentations
    or with the distinct onset of a subject motor action."

would be better as:  "Events are most commonly associated with sudden-onset sensory stimulus presentations
    or with the distinct marker of a subject motor action."

Re:  Many experiments mark both onset and offset and other things.

Reply:
Thanks, I tried to address all of this in bb19602